extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/standalone/VertxRequestHandler.java

```diff
@@ -35,20 +37,25 @@
     protected final Vertx vertx;
     protected final RequestDispatcher dispatcher;
     protected final String servletMappingPrefix;
+    protected final int resteasyUriInfoCacheMaxSize;
     protected final BufferAllocator allocator;
     protected final BeanContainer beanContainer;
     protected final CurrentIdentityAssociation association;
 
+    private final Map<String, ResteasyUriInfo.InitData> resteasyUriInfoInitDataMap = new ConcurrentHashMap<>();;
```

Comment:
Minor nitpick: you have two semi-colon.

Reply:
LOL, I most certainly do, good find! I'll fix it soon

Reply:
Fixed